src/vma/dev/ring_eth_cb.h

```diff
@@ -58,7 +58,8 @@ class ring_eth_cb : public ring_eth
 {
 public:
 	ring_eth_cb(int if_index,
-		    vma_cyclic_buffer_ring_attr *mp_ring, ring *parent = NULL);
+		    vma_cyclic_buffer_ring_attr *mp_ring, iovec *mem_sec = NULL,
+		    ring *parent = NULL);
```

Comment:
please consider the comment https://github.com/Mellanox/libvma/pull/723#discussion_r223632892
to use ring constructor extendable as `ring_eth(int if_index, struct ring_eth_cb_desc *desc, ring *parent = NULL)`
where
```
class ring_eth_cb : public ring_eth
{
public:
	struct ring_eth_cb_desc {
		vma_cyclic_buffer_ring_attr *mp_ring;
		struct iovec *mem_sec;
	};
....
```
and call
```
struct ring_eth_cb::ring_eth_cb_desc desc = {
    &prof->get_desc()->ring_cyclicb,
    key->get_memory_descriptor()
};
ring = new ring_eth_cb(get_if_idx(), &desc);
```